libs/ui-lib/lib/ocm/components/clusterConfiguration/networkConfiguration/VirtualIPControlGroup.tsx

```diff
@@ -108,13 +108,17 @@ export const VirtualIPControlGroup = ({
     (hasDhcp: boolean) => {
       // We need to sync the values back to the form
       setFieldValue('apiVips', hasDhcp ? [] : cluster.apiVips);
+      setFieldTouched('apiVips', true);
```

Comment:
Is `setFieldTouched` necessary?
Would it not work with adding the `shouldValidate` parameter to true in  `setFieldValue`?

eg
```
setFieldValue('apiVips', hasDhcp ? [] : cluster.apiVips, true);
setFieldValue('ingressVips', hasDhcp ? [] : cluster.ingressVips, true);
```

Reply:
true, it works